docs/cli/installation.md

```diff
@@ -18,16 +18,15 @@ $> brew install raito-io/tap/cli
 
 ### Manual installation
 
-Alternatively, go to [the Github releases](https://github.com/raito-io/cli/releases){:target="_blank"}, find the version you want and go from there. E.g.
+If you don't have Brew you can also manually install. To do this, go to [the Github releases](https://github.com/raito-io/cli/releases){:target="_blank"}, find the version you want (the latest is always recommended) and find the build that corresponds to your operating system. Right-click on the file and copy the link URL. 
+The following snippet shows and example of how you can then use that URL to download and unpack the Raito CLI.
 
 ```bash
 $> wget https://github.com/raito-io/cli/releases/download/v0.19.0/raito-0.19.0-darwin_arm64.tar.gz
 $> tar -xzvf raito-0.19.0-darwin_arm64.tar.gz
 $> ./raito --help
 ```
 
-Currently we release for Linux and macOS.
```

Comment:
Is this not true or relevant anymore?

Reply:
It's already in the text above. Felt double